README.adoc

```diff
@@ -8,45 +8,42 @@
-
-You'll also explore the configuration and use of Liberty Messaging Server, along with examples of message production and consumption with Liberty Messaging Server Client. Additionally, you will discover how to use IBM MQ for Jakarta Messaging.
+You’ll learn how to communicate between Java web services when one service is producing a continuous stream of asynchronous messages or events to be consumed by other services, rather than just sending and receiving individual requests for data. You will also learn how to use a messaging server and client to manage the production and consumption of the messages by the services.
 
-In this guide, you will use Jakarta Messaging APIs to build the application and implement a messaging solution that enables communication between different parts of Java microservices. Jakarta Messaging makes it easy to write and configure your application to send, receive, and process the events efficiently.
+In this guide, you will first use the embedded Liberty Messaging Server to manage messages, then you will optionally switch to using an external messaging server to manage the messages, in this case, https://www.ibm.com/products/mq[IBM MQ^]. You might use an external messaging server if the messaging mechanism is important to your application (e.g. during a bank transfer in a banking application). None of your messages are lost if there is a system overloading or outage.
 
-*What is Jakarta Messaging?*
+You will learn how to write your Java application using the Jakarta Messaging API which provides a standard way to produce and consume messages in Java application, regardless of which messaging server your application will ultimately use.
 
-Jakarta Messaging provides an easy way to asynchronously send, receive, and process messages that are received as continuous streams of events. By integrating Jakarta Messaging with Open Liberty, you can easily configure and manage message producers and consumers within your Java microservices. You simply use the Jakarta Messaging API to annotate methods in your application beans, and Open Liberty handles the communication infrastructure, ensuring messages are reliably exchanged. This integration allows your services to connect easily with external messaging systems, such as IBM MQ.
+The application in this guide consists of two microservices, `system` and `inventory`. Every 15 seconds, the `system` microservice computes and publishes a message that contains system’s current CPU and memory load usage. The `inventory` microservice subscribes to that information at the `/systems` REST endpoint so that it can keep an updated list of all the systems and their current system loads.
```

Comment:
```suggestion
The application in this guide consists of two microservices, `system` and `inventory`. Every 15 seconds, the `system` microservice computes and publishes a message that contains the system’s current CPU and memory load usage. The `inventory` microservice subscribes to that information at the `/systems` REST endpoint so that it can keep an updated list of all the systems and their current system loads.
```